### PR TITLE
feature-mamage-job

### PR DESCRIPTION
**Pull request by shawnohn**
image

image

Manage Job page for Sitters

### CODE REVIEW

**Comment by shawnohn** [comment]
FYI, App.js and Navbar.js are sort of temporary since I just fixed some on my local to test upcoming features.
It would be merged into Tongil's version later.

File: client/src/pages/ManageJobPage/BlankComponent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}));
+
+export default function BlankComponent(props) {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root} variant="outlined">
+      <CardContent className={classes.contents}>
+        <Typography variant="h6" gutterBottom>
+          No jobs..
```
Comment (sina-jamshidi) [inline client/src/pages/ManageJobPage/BlankComponent.js:26]: This should say something like "No jobs right now." And maybe something like "Make sure your profile is updated and your availability is set here!" With a link to the edit profile page

File: client/src/pages/SitterDetailPage/SitterDetailPage.js [hunk 2]
```diff
@@ -1,3 +1,382 @@
+// import React, { useState, useEffect } from 'react';
```
Comment (sina-jamshidi) [inline client/src/pages/SitterDetailPage/SitterDetailPage.js]: Lol what happened to this whole file?

File: client/src/pages/ManageJobPage/BlankComponent.js (hunk 1 above)
Comment (shawnohn) [reply client/src/pages/ManageJobPage/BlankComponent.js:26]: Thank you for good ideas!

File: client/src/pages/SitterDetailPage/SitterDetailPage.js (hunk 2 above)
Comment (shawnohn) [reply client/src/pages/SitterDetailPage/SitterDetailPage.js]: Oh, I forget to tell there was some error that new created user's detail isn't loaded well after Tongil's commit.
I simply replaced with my local code for further works and it was committed accidentally.
As it has been fixed, I will pull them and not commit this one.
@TongilKim Please review handling album codes, it seems need to consider when user doesn't have albums.

Comment (sina-jamshidi) [reply client/src/pages/SitterDetailPage/SitterDetailPage.js]: Ah okay thanks!

Comment (TongilKim) [reply client/src/pages/SitterDetailPage/SitterDetailPage.js]: Yes, I fixed that. sorry for the inconvenience.

**Pull request merged**